docs/getting_started.md

```diff
@@ -151,7 +151,7 @@ Sep 30 09:36:19 cga-centreon-19-10.int.centreon.com systemd[1]: Started Centreon
 If you are using the sources, execute the following command:
 
 ```bash
-perl gorgoned --config=config/gorgoned.yml --severity=error
+perl gorgoned --config=config/config.yaml --severity=error

```

Comment:
@garnier-quentin , please check the consistency of this line

Reply:
gorgoned can use that file (if the file is well formatted)

Reply:
isn't more like : config=/etc/centreon-gorgone/config.yaml ? like on line 137

Reply:
it's for source